packages/svelte/src/compiler/phases/2-analyze/css/css-analyze.js

```diff
@@ -133,7 +133,7 @@ const css_visitors = {
 
 		node.metadata.is_global = node.selectors.length >= 1 && is_global(node);
 
-		if (node.selectors.length === 1) {
+		if (node.selectors.length >= 1) {
```

Comment:
if we narrow it like this it should be fine
```suggestion
		if (
			node.selectors.every(
				(selector) =>
					selector.type === 'PseudoClassSelector' || selector.type === 'PseudoElementSelector'
			)
		) {
```